roles/awsenv/tasks/destroy_vpc.yml

```diff
@@ -66,6 +66,7 @@
     - name: Destroy route53 zone records
       shell: aws route53 change-resource-record-sets --hosted-zone-id "{{ internal_zone_id }}" --change-batch "file://{{ playbook_dir }}/route53-delete.json"
       changed_when: false
+      when: dns_zone_records.ResourceRecordSets.ResourceRecords|default([]) |count > 4
```

Comment:
i missed that. it should be two, as the soa and ns records don't stop the zone from being deleted